gc/gc-tool/src/test/java/org/projectnessie/gc/tool/cli/commands/TestMarkOptions.java

```diff
@@ -148,6 +148,7 @@ public void cutoffPoliciesPerRef() {
         .isEqualTo(CutoffPolicy.numCommits(1));
   }
 
+  @SuppressWarnings("TypeParameterUnusedInFormals")
```

Comment:
these are errorprone (not checkstyle) warnings:
http://errorprone.info/bugpattern/TypeParameterUnusedInFormals

wondering: couldnt we have fixed this as suggested?

Reply:
err - yes